src/main/java/picocli/CommandLine.java

```diff
@@ -4886,6 +4886,30 @@ private static String format(String formatString, Object... params) {
          */
         String[] getVersion() throws Exception;
     }
+
+    /**
+     * Converter which can be used in case when picocli should use default convert. For example this can be used in maps:
+     *
+     * <pre>
+     *   class App {
+     *       &#064;Option(names = "-D", converter = {UseDefaultConverter.class, GenericValueConverter.class})
+     *       Map&lt;String, GenericValue&lt;?&gt;&gt; values;
+     *  }
+     * </pre>
+     *
+     * Instances of this class will throw UnsupportedOperationException for {@link #convert(String)} method.
+     */
+    public static final class UseDefaultConverter implements ITypeConverter<Object> {
```

Comment:
`UseDefaultConverter` is only for use cases when we need to provide converters for multiple parameters and for some of them we simple want to use default, e.g.:

```java
           @Option(names = "-D", converter = {UseDefaultConverter.class, GenericValueConverter.class})
            Map<String, GenericValue<?>> values;
```

In this case I would need to provided custom convert for String class also since `BuiltIt.StringConverter` is private.

Reply:
I understand that. Is there any issue with using the `CommandLine::registerConverter` method to register the `GenericValueConverter`?

Reply:
With global registration there may be issue with type erasure. If I want to use two different converters for same `GenericValue<T>` (e.g. in one parameter `GenericValue<Integer> and in other `GenericValue<String>`) then I won't be able to this with `CommandLine::registerConverter`